### ISSUE TITLE
staking but not showing ?

### ISSUE THREAD

**Issue by mahalamobile** [qtumproject/qtum#926]
so this shows im staking, wallet is encrypted and unlocked for staking

ubuntu@blockchain:~/qtum/src$ ./qtum-cli getstakinginfo
{
  "enabled": true,
  "staking": true,
  "errors": "",
  "currentblocktx": 0,
  "pooledtx": 1,
  "difficulty": 5702714.96436759,
  "search-interval": 162609,
  "weight": 28832500000,
  "delegateweight": 0,
  "netstakeweight": 2646958805433621,
  "expectedtime": 11751000
}

but then 
ubuntu@blockchain:~/qtum/src$ ./qtum-cli getwalletinfo                          {
  "walletname": "",
  "walletversion": 169900,
  "balance": 305.24100000,
  "stake": 0.00000000,
  "unconfirmed_balance": 0.00000000,
  "immature_balance": 0.00000000,
  "txcount": 6,
  "keypoololdest": 1618237532,
  "keypoolsize": 1000,
  "hdseedid": "8af54f4563643b5e81b0ebd782ce2e04648c1f16",
  "keypoolsize_hd_internal": 1000,
  "unlocked_until": 1718476436,
  "paytxfee": 0.00000000,
  "private_keys_enabled": true,
  "avoid_reuse": false,
  "scanning": false
}
show my stake to be 0.000000, where my balance is 305.24100000

this command shows nada

ubuntu@blockchain:~/qtum/src$ ./qtum-cli getdelegationsforstaker "QLeKxnZg5KDSq5422jgx4VbWMyXq8ycdu4"
[
]

and have run this
ubuntu@blockchain:~/qtum/src$ ./qtum-cli splitutxosforaddress "QLeKxnZg5KDSq5422jgx4VbWMyXq8ycdu4" 100 200
{
  "selected": "34.412",
  "splited": "0.00"
}

PLEASE HELP :)

**Comment by mahalamobile** [comment]
? is anyone still working on this project? thanks again

**Comment by JB395** [comment]
Yes, read about the technology here https://github.com/qtumproject/documents/blob/master/en/SUMMARY.md

Your staker is setup correctly, staking 305 QTUM your expected time to a block reward is about 3 months https://qtum.info/misc/toolbox/stake-calculator

Good luck! Tech support at https://t.me/qtumofficial

**Comment by mahalamobile** [comment]
great thanks appreciated.  when u do find a block reward does this get
added to your balance or goes into a reward wallet? in other words where
does the payout go? Thanks again

**Comment by JB395** [comment]
The block reward goes immediately to the staking wallet address, and matures after 2,000 confirmations (now). Basically the block publishing wallet sends a transaction to itself committing the stake for 2,000 confirmations. For example, the wallet could use a UTXO with 100 QTUM as the input for the stake and receive an output of 101 QTUM (plus some transaction fees and gas, if in that block), thus paying itself the 1.0 QTUM block reward.

**Comment by mahalamobile** [comment]
great thanks for the explaining

i read that now a block reward is 1 qtum but 4 times as many blocks get
produced, is correct?

also i see the reward  calculator went from roughly 5% apy to like 16%. is
that accurate?

Regards

**Comment by JB395** [comment]
Yes, previously the subsidy was 4.0 QTUM for 675 blocks/day or 2,700 QTUM/day.
After the hard fork the subsidy is 1.0 QTUM for 2,700 blocks/day or 2,700 QTUM/day, so inflation stays the same.

The qtum.info stake calculator [https://qtum.info/misc/toolbox/stake-calculator](https://qtum.info/misc/toolbox/stake-calculator) is correct. Return is based on network weight, which changes over time.
